climada/hazard/trop_cyclone.py

```diff
@@ -472,8 +485,8 @@ def compute_windfields(track, centroids, model):
     v_full = v_trans[1][1:, None, :] * v_trans_corr[1:, :, None] + v_ang
     v_full[np.isnan(v_full)] = 0
 
-    windfields[1:, track_centr_idx, :] = v_full
-    return windfields
+    windfields[1:, :, :] = v_full
+    return windfields, reachable_centroids
 
 def _close_centroids(t_lat, t_lon, centroids):
```

Comment:
Should the bounding box size maybe be a variable?

Reply:
Since the bounding box is not introduced in this PR, I think changing it is beyond the scope of this PR. We should discuss this separately. The bounding box was there before to determine the centroids on which computations are performed. I only made sure that the memory allocation is restricted to those centroids for which computations will be performed.